### DISCUSSION TITLE
How to use own template for each of the pages created in control panel?

### DISCUSSION THREAD

**Discussion by pradeepkr1303** [statamic/cms#4964] [Q&A] [answered]
Hi all,

I created a statamic app using studio1902/statamic-peak starter kit. I created the pages which I needed and added some content to each page using the inbuilt page builder. When I visited the page in application, the dom renders the default html design by the page builder.

The question are:

1. how I can use my own html design for the pages I created in control panel?
2. how to add css stylings for the template? (scss, if possible)
3. for each page there is a markdown file. How can I use the data in markdown file in the template?

![image](https://user-images.githubusercontent.com/25196812/147280899-ac571aef-0b28-44b9-bacb-eefb514e88a0.png)

Thanks in advance!

**Comment by jackmcdade** [comment, marked as answer]
Hey there! All of these questions are the basic "How to Use Statamic 101" things, none of which are particularly exclusive to the starter kit. I **highly recommend** spending a half hour or so reading the [Quick Start Guide](https://statamic.dev/quick-start-guide) or watching the first view [Statamic 3 Walkthrough Guide videos](https://www.youtube.com/playlist?list=PLVZTm2PNrzMwYLGotkQvTvjsXAkANJIkc) and all of these questions (and more) will be answered :)
